index.html

```diff
@@ -284,34 +313,37 @@ <h5 class="mb-0">
     
     
         <div class="d-header" id="headingThree">
-            <h5 class="mb-0">
-                <button class="btn collapsed" type="button" data-toggle="collapse" data-target="#collapseThree" aria-expanded="false" aria-controls="collapseThree">
-                Skills
-                </button>
+            <h5 class="skill-title">
+                 Skills
             </h5>
+            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="skill-arrow bi-chevron-compact-down" viewBox="0 0 16 16">
+                <path fill-rule="evenodd" d="M1.553 6.776a.5.5 0 0 1 .67-.223L8 9.44l5.776-2.888a.5.5 0 1 1 .448.894l-6 3a.5.5 0 0 1-.448 0l-6-3a.5.5 0 0 1-.223-.67z"/>
+              </svg>
         </div>
             <div id="collapseThree" class="collapse" aria-labelledby="headingThree" data-parent="#accordionExample">
             <div class="card-body">
                 
             </div>
         </div>
         
-        </div>
+    </div>
     </section>
     <footer class="contact-me">
-        <h3 class=contact-head>Contact me</h3>
-        <p class="contact-description">
-            If you have an application you are interested 
-            in developing, a feature that you need built 
-            or a project that needs coding. I’d love to 
-            help with it 
-        </p>
-        <form id="form" action="https://formspree.io/f/1" method="POST">
-            <input type="text" name="name" id="name" class="name-email" placeholder="Enter your name" maxlength="30" required>
-            <input type="email" name="email" id="email" class="name-email" placeholder="Enter your email" required>
-            <textarea name="message" id="message" rows="6" maxlength="500" placeholder="write your message here" required></textarea>
-            <button type="submit" class="button" >Get it Touch</button>
-        </form>
+        <div class="contact-me-div">
+            <h3 class=contact-head>Contact me</h3>
+            <p class="contact-description">
+                If you have an application you are interested 
+                in developing, a feature that you need built 
+                or a project that needs coding. I’d love to 
+                help with it 
+            </p>
+            <form id="form" action="https://formspree.io/f/1" method="POST">
+                <input type="text" name="name" id="name" class="name-email" placeholder="Enter your name" maxlength="30" required>
+                <input type="email" name="email" id="email" class="name-email" placeholder="Enter your email" required>
+                <textarea name="message" id="message" rows="6" maxlength="500" placeholder="write your message here" required></textarea>
+                <button type="submit" class="button" >Get it Touch</button>
+            </form>
+        </div>
```

Comment:
I think all the elements of the form are centered in the page, including the `Get in touch` button :blush:

Reply:
Nailed this👍